src/Stripe.net/Services/_interfaces/ISingletonRetrievable.cs

```diff
@@ -3,6 +3,8 @@ namespace Stripe
     using System.Threading;
     using System.Threading.Tasks;
 
+    // TODO (MAJOR): Accept a TOptions generic and use it to define method overloads
+    // that allow passing in options.
```

Comment:
I'm thinking we should update this TODO so that we aren't defining overloads, but rather replacing the existing `Get` methods with those that accept both options and request options.
```suggestion
    // TODO (MAJOR): Accept a TOptions generic and use it to define methods
    // that accept options parameters.
```